x-pack/plugins/apm/public/components/alerting/fields.test.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { ServiceField, TransactionTypeField } from './fields';
+import { render } from '@testing-library/react';
+import { expectTextsInDocument } from '../../utils/testHelpers';
+
+describe('alerting fields', () => {
+  describe('Service Fiels', () => {
+    it('renders with value', () => {
+      const component = render(<ServiceField value="foo" />);
+      expectTextsInDocument(component, ['foo']);
+    });
+    it('renders with All when value is not defined', () => {
+      const component = render(<ServiceField />);
+      expectTextsInDocument(component, ['All']);
+    });
+  });
+  describe('Transaction Type Field', () => {
+    it('renders select field when multiple options available', () => {
+      const options = [
+        { text: 'Foo', value: 'foo' },
+        { text: 'Bar', value: 'bar' },
+      ];
+      const component = render(
+        <TransactionTypeField currentValue="Foo" options={options} />
+      );
+      expectTextsInDocument(component, ['Foo']);
+    });
```

Comment:
Does this test actually verify that it's possible to select "Bar"? Couldn't it be read-only of "Foo" and still pass?